playbooks/create_configmap_and_secret.yaml

```diff
@@ -0,0 +1,29 @@
+---
+- name: Playbook to create Thoth ConfigMap and Secret
+  tags:
+    - openshift
+    - thoth
+
+  hosts: localhost
+  gather_facts: false
+  connection: local
+
+  roles:
+    - role: thoth-station.configmaps_and_secrets
```

Comment:
So we will track changes in the `configmaps-and-secrets` repo? How will they be updated?

Reply:
ja, that role will deploy the configmap/secret template, the templates themselves are created via core_templates role, and the template files themselves still life in each repository.

Reply:
How do we update them on galaxy?

Reply:
the roles? they are repositories to be importet into ansible-galaxy... `ansible-galaxy import thoth-station core-imagestreams`